Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
```

Comment:
I know that some people only know of the modulo operator as 'mod' and not '%' so maybe we can add or operation == 'mod' and specify that in the instruction message.

Reply:
I agree with that. I learnt that many people grew up with the modulo operator as 'mod' and not '%'